compiler/src/dmd/mustuse.d

```diff
@@ -18,17 +18,6 @@ import dmd.globals;
 import dmd.identifier;
 import dmd.location;
 
-// Used in isIncrementOrDecrement
```

Comment:
all the changes in this file look unrelated.

Reply:
As mentioned in the opening comment, this was to fix a cyclic static constructor dependency induced by this reorganization.

Reply:
Because `Loc.initial` is just `Loc(0,0,0)`, maybe add a ctor for StringExp _without_ `const ref Loc`?